CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+* **experimental**: Retrieve sessions/session actions for auto download ([`5fdc5bf`](https://github.com/aws-deadline/deadline-cloud/commit/5fdc5bf7ce03599286d48cbbfdf6f4701724dec7))
+* Remember last used priority, max retries per task, max failed t… (#711) ([`358e6c7`](https://github.com/aws-deadline/deadline-cloud/commit/358e6c7d4fa2cf194d77c400803415eeb4dd5ffa))
+* **experimental**: Track jobs list for incremental download over time ([`9a9d59e`](https://github.com/aws-deadline/deadline-cloud/commit/9a9d59edc6474410452bbb3ed9b1f860c25d96a6))
+* **cli**: Add job 'wait' and 'logs' API/CLI commands (#697) ([`8ea699f`](https://github.com/aws-deadline/deadline-cloud/commit/8ea699f1a28d7b3ebe4ad31573e72b5380af21fe))
+* **experimental**: Add client library API to list jobs by filter expr ([`93c71a6`](https://github.com/aws-deadline/deadline-cloud/commit/93c71a6ed266197317cdc18f9c3d0f550963e4e0))
+* **experimental**: Refine the IncrementalDownloadState class ([`8c922de`](https://github.com/aws-deadline/deadline-cloud/commit/8c922debfc3b30317cf86ae310c53ef594cedb2c))
+* **experimental**: Add client library API to list recently updated jobs ([`bd32564`](https://github.com/aws-deadline/deadline-cloud/commit/bd325640e573bfabc8865b8a91f6158a733c7e75))
+* **experimental**: command bootstrap logic and reading from, saving to state file for incremental downloads cli (#691) ([`7acfcba`](https://github.com/aws-deadline/deadline-cloud/commit/7acfcba3271e633c81c0887d1a049ebd933a97d0))
+* **experimental**: command bootstrap logic and reading from, saving to state file for incremental downloads cli ([`7acfcba`](https://github.com/aws-deadline/deadline-cloud/commit/7acfcba3271e633c81c0887d1a049ebd933a97d0))
+* **experimental**: command bootstrap logic and reading from, saving to state file for incremental downloads cli (#691) ([`b0488a6`](https://github.com/aws-deadline/deadline-cloud/commit/b0488a62bad8927b888dff0f0617a9cbc477d729))
+* **experimental**: command bootstrap logic and reading from, saving to state file for incremental downloads cli ([`b0488a6`](https://github.com/aws-deadline/deadline-cloud/commit/b0488a62bad8927b888dff0f0617a9cbc477d729))
+* Adding a helper to initialize the lru_cached get_s3_client method. Adding a cached get_session_client method to allow get_boto3_client calls which end up using the same session and same service name to use cached clients. (#708) ([`5665fa8`](https://github.com/aws-deadline/deadline-cloud/commit/5665fa8813ef8bb5c01a57c9604bbaed7015d859))
+* **experimental**: fix atomicity for pid lock & add release lock support for incremental downloads cli (#682) ([`da853ac`](https://github.com/aws-deadline/deadline-cloud/commit/da853ac0b5e1d15e7e2c60d908e71f81d0b39521))
+* **experimental**: pid locking atomicity fixes for automatic incremental downloads ([`da853ac`](https://github.com/aws-deadline/deadline-cloud/commit/da853ac0b5e1d15e7e2c60d908e71f81d0b39521))
```

Comment:
There's a few duplicate messages in here

Can we also group Experimental changes in a separate section? Here's an example of what it can look like: https://github.com/aws-deadline/deadline-cloud-worker-agent/releases/tag/0.28.10

Reply:
While we do this, would like if we can make some of these feature changes more descriptive as to what it provides. 

For example:

`command bootstrap logic and reading from, saving to state file for incremental downloads cli ` is really unclear